packages/website/docs/04-services-and-layers.md

```diff
@@ -422,10 +416,10 @@ class Counter extends Context.Tag("@app/Counter")<
 >() {
   static readonly layer = Layer.sync(Counter, () => {
     let count = 0
-    return {
+    return Counter.of({
```

Comment:
Ah, but here we're adding Counter.of?
I don't think I have a pref strong preference either way, whichever one gives us the best type inference. But we should probably be consistent.